### PR TITLE
[stable/odoo] Update PostgreSQL as deps

### PR DESCRIPTION
**Pull request by carrodher**
Update Odoo dependencies to include the new PostgreSQL version

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

**Review by juan131 — APPROVED** [review]
LGTM

File: stable/odoo/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: odoo
-version: 3.2.2
+version: 3.2.3
```
Comment (tompizmor) [inline stable/odoo/Chart.yaml]:
```suggestion
version: 4.0.0
```

As there is a big change in the dependency I think we should also update the major version in the parent chart.

**Review by tompizmor — APPROVED** [review]
/lgtm

**Pull request merged**